base/pre_hook_rocm_smi_lib.cmake

```diff
@@ -1 +1 @@
-set(CMAKE_INSTALL_RPATH "$ORIGIN;$ORIGIN/llvm/lib;$ORIGIN/rocm_sysdeps/lib")
+set(CMAKE_INSTALL_RPATH "$ORIGIN;$ORIGIN/llvm/lib;$ORIGIN/lib")
```

Comment:
I think it needs an RPATH to rocm_sysdeps so it can find libdrm.

Reply:
Ah yeah, for testing I build without bundled sysdeps and therefore assumed this might not be needed. I bring back the original line. Thanks for spotting this.